packs/panther.yml

```diff
@@ -7,6 +7,10 @@ PackDefinition:
     - Panther.SAML.Modified
     - Panther.Sensitive.Role
     - Panther.User.Modified
+    - IOC.SunburstFQDNIOCs
```

Comment:
All of these IOC/0DayIP rules need to be deprecated, they are pretty stale at this point.  Also the IOC.Log4jExploit rule I believe causes severe performance issues.